### PR TITLE
VY 105 Add Employee Detail View

### PR DESCRIPTION
**Pull request by lf-romitamgai**
@lf-kiranpariyar @joshianuj Please review the code

### CODE REVIEW

**Comment by bhuwang** [comment]
Can one of the admins verify this patch?

**Comment by bhuwang** [comment]
Can one of the admins verify this patch?

File: frontend/src/js/components/employee/employee-view/SocialAccountsInfo.js [hunk 1]
```diff
@@ -14,27 +18,27 @@ class SocialAccountsInfo extends React.Component{
   }
 
   renderSocialAccountList(account,key) {
-    let accountExtension = account.split('@');
+    const socialAccountTitle = account.socialAccountType.title.toLowerCase();
     let fontClass = '';
 
-    if(accountExtension[1] == 'skype.com') {
+    if(socialAccountTitle== 'skype') {
```
Comment (joshianuj) [inline frontend/src/js/components/employee/employee-view/SocialAccountsInfo.js]: use switch and add default case

File: frontend/src/js/components/employee/employee-view/employee-tabbed-view/EmployeeInfoTab.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    }
+  }
+
+  //TODO
+  handleSelect(index, last) {
+  }
+
+  toggleHRInfoFormView() {
+    this.setState({
+      displayHRInfoView: !this.state.displayHRInfoView,
```
Comment (joshianuj) [inline frontend/src/js/components/employee/employee-view/employee-tabbed-view/EmployeeInfoTab.js]: Refactor

File: frontend/src/js/utils/DateUtil.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+/**
+ * Created by romit on 6/8/16.
+ * Romit Amgai <romitamgai@lftechnology.com>
+ * 6/8/16
+ */
+
+class DateUtil {
+
+  static formatDate(date) {
```
Comment (joshianuj) [inline frontend/src/js/utils/DateUtil.js]: documentation

File: frontend/src/js/components/employee/EmployeeRow.js [hunk 4]
```diff
@@ -17,25 +17,33 @@ import StringUtil from'../../utils/StringUtil';
 
 class EmployeeRow extends React.Component {
 
-  constructor(props) {
-    super(props);
-    this.stringUtil = new StringUtil();
+  renderPhoneNumbers(phones) {
```
Comment (joshianuj) [inline frontend/src/js/components/employee/EmployeeRow.js]: move this to util

**Pull request merged**